WireNetwork/Sources/WireNetwork/APIs/Rest/AuthenticationAPI/AuthenticationAPIV0.swift

```diff
@@ -271,9 +270,9 @@ class AuthenticationAPIV0: AuthenticationAPI, VersionedAPI {
             .withBody(body, contentType: .json)
             .build()
 
-        let (data, response) = try await networkService.executeRequest(request)
+        let (_, _) = try await networkService.executeRequest(request)
 
-        try ResponseParser()
+        _ = ResponseParser()
```

Comment:
issue: so here we don't do anything if at least the response is not passed to the parser